go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/mongodb/mongodb-atlas-kubernetes/v2
 
 go 1.22.0
 
-toolchain go1.22.4
+toolchain go1.22.5
```

Comment:
we have other go.mod files in the repo as well. Please update those as well.

Reply:
Done